cmd/horcrux/cmd/key2shares.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+
+	"github.com/jackzampolin/horcrux/internal/signer"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	tmOS "github.com/tendermint/tendermint/libs/os"
+	"github.com/tendermint/tendermint/privval"
+	tsed25519 "gitlab.com/polychainlabs/threshold-ed25519/pkg"
+)
+
+// key2sharesCmd represents the key2shares command
+var key2sharesCmd = &cobra.Command{
```

Comment:
Currently this command will dump the key shares in the current directory. Maybe we want to dump them in the `~/.horcux/shares` dir or something? Current behavior is OK but maybe there is something better?